### ISSUE TITLE
An application should not map to a project

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-developer/ocdev#83]
Looking at the output and source code, it looks like an application is mapped to a project.

```
$ oc get project
NAME                  DISPLAY NAME   STATUS
devconfcz18                          Active
labs                                 Active
samplelabs                           Active
the-lounge-jmorales                  Active

$ ocdev application list
devconfcz18
labs
samplelabs
the-lounge-jmorales
```

As can be inferred from "Use case 14: As a developer, I want to list all applications in my project" an OpenShift project can have multiple applications.

**Comment by kadel** [comment]
mapping applications to project is a temporary simplification because we didn't wont to get blocked on this at the beginning.

**Comment by kadel** [comment]
closed in #94 
there are still some issues with it, separate issues were created to address those #105 #104
